### DISCUSSION TITLE
Disponibilité de la Criminalité sur l'OSS.

### DISCUSSION THREAD

**Discussion by KevinJ-Dev** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#319] [Q&A] [answered]
Bonsoir,

J'aurais voulu savoir si la partie criminalité est t-il open source ? car je ne trouve pas ou il se trouve? Je vois bien qu'il y a des items illégaux, mais pas le reste

**Comment by Laikker** [comment, marked as answer]
Hello,

Non elle ne l'est pas

**Reply by Rotabla42** [reply]
Hello, 
C'est quelque chose qui est prévu au programme ou pas du tout ?

**Comment by NariieL** [comment]
Coucou ! Je te confirme que rien n'est prévu.

> La Moyenne Criminalité, la Pêche, la Drogue, la Police Scientifique et la Haute Criminalité ne seront pas disponible sur l'OSS.

Le rôle de l'OSS est d'offrir une base extrêmement dense d'un Serveur FiveM lié à une centaine de features entièrement maison. Cela suffit amplement pour tout créateur souhaitant lancer son propre serveur, tout en lui permettant d'enrichir son serveur avec sa propre vision de la criminalité.

D'autant plus que nous préférons conserver notre authenticités tant que SOZ vie afin d'assurer la stabilité du serveur. 

Peut être que lorsque le serveur fermera, tout sera disponible ! <3
